### PR TITLE
#13408: Pytorch tracing sweeps - eltwise

### PR DESCRIPTION
**Pull request by KalaivaniMCW**
### Ticket
Link to Github Issue #13408

### Problem description
Need sweep tests for Pytorch tracing

### What's changed
- Added sweeps for following ops 
image

Also, 
- Added golden function for `ttnn.elu` with `alpha` kw-arg param
- Changed kw-arg name for `ttnn.leaky_relu` to match pytorch api

Note: some commented code is left in certain sweep files; it will be removed once we are done with identifying root cause of all fails in that file.

### Checklist
- [x] Post commit CI https://github.com/tenstorrent/tt-metal/actions/runs/11218528419
https://github.com/tenstorrent/tt-metal/actions/runs/11266241044
- [ ] Blackhole Post commit (if applicable)
- [ ] Model regression CI testing passes (if applicable)
- [ ] Device performance regression CI testing passes (if applicable)
- [x] New/Existing tests provide coverage for changes

Sweep test Results Tracker: [Link](https://docs.google.com/spreadsheets/d/1zumGqgcrweLBY4wn1OzKCcf-DJDSiaj6_bDvjfT74JY/edit?usp=sharing)

Sweeps: single test
https://github.com/tenstorrent/tt-metal/actions/runs/11218584287
all - https://github.com/tenstorrent/tt-metal/actions/runs/11223485745

Summary : https://docs.google.com/spreadsheets/d/1l9DscsVWUfKjPz1zVJ0PLNhODRQJyjYi6bLGnRTGZFw/edit?usp=sharing

### CODE REVIEW

**Review by Aswinmcw — APPROVED** [review]
LGTM

**Comment by sjameelTT** [comment]
Did you run all the sweeps?

**Comment by KalaivaniMCW** [comment]
> Did you run all the sweeps on CI? Do you have a link to the combined run for nightly? I'm wondering about the time.
@sjameelTT 
I did run a single sweep https://github.com/tenstorrent/tt-metal/actions/runs/11218584287 which is taking about 2 hrs. (in my local it takes only a few seconds though). i've triggered a full ttnn nightly sweeps as well.

File: tests/sweep_framework/sweeps/eltwise/binary/eq/eq_scalar_pytorch2.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+) -> list:
+    data_seed = random.randint(0, 20000000)
+    torch.manual_seed(data_seed)
+
+    torch_input_tensor_a = gen_func_with_cast_tt(
+        partial(torch_random, low=-100, high=100, dtype=torch.float32), input_a_dtype
+    )(input_shape)
+
+    torch_output_tensor = torch.eq(torch_input_tensor_a, scalar)
+
```
Comment (eyonland) [inline tests/sweep_framework/sweeps/eltwise/binary/eq/eq_scalar_pytorch2.py:74]: Are the zero dim tensor with scalar values coming on another PR?

**Review by VirdhatchaniKN — APPROVED** [review]

File: tests/sweep_framework/sweeps/eltwise/binary/eq/eq_scalar_pytorch2.py (hunk 1 above)
Comment (KalaivaniMCW) [reply tests/sweep_framework/sweeps/eltwise/binary/eq/eq_scalar_pytorch2.py:74]: In this case, for eq the scalar values are not passed as Tensor but a number https://github.com/tenstorrent/pytorch2.0_ttnn/blob/main/docs/operations/aten.eq.Scalar.md

**Review by eyonland — APPROVED** [review]

**Review by mcw-anasuya — APPROVED** [review]

**Review by sjameelTT — APPROVED** [review]

**Review by jdesousa-TT — APPROVED** [review]

**Review by ayerofieiev-tt — APPROVED** [review]

**Pull request merged**